include/slang/ast/symbols/InstanceSymbols.h

```diff
@@ -207,6 +208,10 @@ class SLANG_EXPORT InstanceBodySymbol : public Symbol, public Scope {
     const DefinitionSymbol& definition;
     mutable std::span<const Symbol* const> portList;
     std::span<const ParameterSymbolBase* const> parameters;
+    // Store any net alias bits to check it's possible intersections.
+    // Since alias statements can appear anywhere module instance statements can appears
+    // good option is to store aliased bits info in the module instance scope.
+    mutable AliasMap* aliasedBits;
```

Comment:
It seems unnecessary to have an alias map allocated for every instance. The rules about net alias overlap are global, so it seems fine to have just one alias map stored in the Compilation object that all aliases can access. Then you don't need a custom allocator for it either.

Reply:
reimplemented all in order to last commentary